python/cudf/tests/test_csvreader.py

```diff
@@ -115,6 +115,22 @@ def test_csv_reader_mixed_data_delimiter(tmpdir):
     assert len(out.columns) == len(df_out.columns)
 
 
+def test_csv_reader_mixed_data_sep(tmpdir):
+
+    fname = tmpdir.mkdir("gdf_csv").join('tmp_csvreader_file3.csv')
```

Comment:
This filename is the same as the previous `test_csv_reader_mixed_data_delimiter()`.

Actually since the majority of the test is the same, can we combine the two and `@pytest.mark.parametrize` sep and delimiter?

Reply:
Are you suggesting that I should combine `test_csv_reader_mixed_data_delimiter` and ` test_csv_reader_mixed_data_sep` togather? 
In `test_csv_reader_mixed_data_delimiter`, we are comparing cudf's`delimiter` with pandas's`delimiter` and in the test `test_csv_reader_mixed_data_sep`, we are comparing the cudf's `delimiter` and cudf's`sep` arg.

Using `@pytest.mark.parametrize`, we can parametrize the value of some variable. Can we parametrize the argument name also? Since I haven't use it yet.

Reply:
Technically, you can parameterize all or any of the arguments since it's Python. One quick idea is for three or four parameters with different names (ex. pandas_sep, pandas_delim, cudf_sep, cudf_delim) and then you specify any combination of them...

Reply:
You can define a dictionary of argument-->value and then pass it into the call by unpacking it:
i.e.
```python
args = {'sep': '|', 'dayfirst': True}
read_csv(fname, names=['1', '2'], dtype=['int64', 'int64'], **args)
```